corehq/apps/tzmigration/planning.py

```diff
@@ -0,0 +1,198 @@
+import sqlite3
+import json
+
+from sqlite3 import dbapi2 as sqlite
+from sqlalchemy import create_engine, Column, Integer, ForeignKey, String, \
+    UnicodeText, Text
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm.exc import NoResultFound
+
+Base = declarative_base()
+
+
+class PlanningForm(Base):
```

Comment:
Yeah I was initially just writing SQLite queries using the `sqlite3` module, but that was feeling pretty ridiculous.